app/brave_generated_resources.grd

```diff
@@ -466,23 +466,6 @@ Or change later at <ph name="SETTINGS_EXTENIONS_LINK">$2<ex>brave://settings/ext
         To disable this extension in private mode and Tor mode, unselect this option.
       </message>
 
-      <!-- In-page Translation -->
-      <message name="IDS_BRAVE_TRANSLATE_BUBBLE_BEFORE_TRANSLATE_TITLE" desc="Title text for the translate bubble when asking to translate a page.">
```

Comment:
This key is replaced to chromium `IDS_BRAVE_TRANSLATE_BUBBLE_BEFORE_TRANSLATE_TITLE`:  `Translate this page?`
https://github.com/brave/brave-core/blob/master/app/generated_resources.grd#L9916

It's used only as the bubble title and invisible for UI (I believe it's used in accessibility mode)